### ISSUE TITLE
Could not find a version that satisfies the requirement dataclasses==0.8

### ISSUE THREAD

**Issue by iamtodor** [ericvsmith/dataclasses#165]
Hello guys,

Somehow I cannot install the latest available version 0.8
I can install only 0.6
What am I missing? What goes wrong in my setup?

Python 3.8
Pip 21.0.01

```
(venv)  usr@macbek ~/programming/code/project pip install dataclasses==0.8
ERROR: Could not find a version that satisfies the requirement dataclasses==0.8
ERROR: No matching distribution found for dataclasses==0.8
(venv)  usr@macbek ~/programming/code/project pip --version
pip 21.0.1 from /Users/usr/programming/code/project/venv/lib/python3.8/site-packages/pip (python 3.8)
```

**Comment by ericvsmith** [comment]
This is not the right place to ask questions about versions of python > 3.6, For those, use the python bug tracker if you think you've found a bug. This repo is only about the 3.6 backport.

dataclasses is built in to python 3.8, so you don't need to install anything, you can just use it. Enjoy!

**Comment by iamtodor** [comment]
Fair enough :)
Thank you @ericvsmith

**Comment by imkaka** [comment]
Hey Eric, 
The latest we can download is  `0.6` when Pypi says `0.8` 

```
❯ pip install dataclasses
Collecting dataclasses
  Downloading dataclasses-0.6-py3-none-any.whl (14 kB)
Installing collected packages: dataclasses
Successfully installed dataclasses-0.6
```

It is breaking as we have pinned `0.8` in requirements.
```
ERROR: Could not find a version that satisfies the requirement dataclasses==0.8
ERROR: No matching distribution found for dataclasses==0.8

**Comment by ericvsmith** [comment]
@imkaka : I'm not really able to help you debug pip issues, but I'll see what I can do. 

What version of python are you using `python -V`? Also, try `python -m pip install dataclasses` to make sure you're using the same version of pip.

**Comment by someq** [comment]
Had the same issue, resolved it after manually installing python 3.6 and creating virtualenv with it. My default python was 3.8 and it did not show this package versions higher then 0.6.

**Comment by jstasiak** [comment]
You can't use dataclasses from PyPI on Python 3.7+ since it's built in there, you can only use the included version. You can have a conditional dependency using https://www.python.org/dev/peps/pep-0345/#environment-markers
